virt/live_migration/virt-migrating-vm-on-secondary-network.adoc

```diff
@@ -1,12 +1,12 @@
 :_content-type: ASSEMBLY
 [id="virt-migrating-vm-on-secondary-network"]
-= Migrating a virtual machine over a dedicated secondary network
+= Migrating a virtual machine over a dedicated additional network
 include::_attributes/common-attributes.adoc[]
 :context: virt-migrating-vm-on-secondary-network
 
 toc::[]
 
-You can configure a dedicated xref:../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[secondary Multus network] for live migration. A dedicated network minimizes disruption to tenant workloads due to network saturation when virtual machine live migration is triggered.
+You can configure a dedicated xref:../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[Multus network] for live migration. A dedicated network minimizes disruption to tenant workloads due to network saturation when virtual machine live migration is triggered.
```

Comment:
That second sentence has some misplaced clauses. Suggestion: "A dedicated network minimizes the effects of network saturation on tenant workloads during live migration."

Reply:
Done.